examples/trossen_ai/main.py

```diff
@@ -0,0 +1,255 @@
+            [-np.pi, np.pi],           # left_joint_5.pos
+            [0.0, 0.04],               # left_left_carriage_joint.pos
+            [-np.pi, np.pi],           # right_joint_0.pos
+            [-np.pi / 2, np.pi / 2],  # right_joint_1.pos
+            [-np.pi / 2, np.pi / 2],  # right_joint_2.pos
+            [-np.pi, np.pi],           # right_joint_3.pos
+            [-np.pi / 2, np.pi / 2],  # right_joint_4.pos
+            [-np.pi, np.pi],           # right_joint_5.pos
+            [0.0, 0.04],               # right_left_carriage_joint.pos
+        ])
+        for i in range(len(full_action)):
+            full_action[i] = np.clip(full_action[i], joint_limits[i][0], joint_limits[i][1])
+
+        if self.test_mode:
+            logger.info(f"TEST MODE: Would execute action: {full_action}")
+            return
+
+        left_arm_action = full_action[:7]
+        right_arm_action = full_action[7:]
+        action_dict = {}
+        for i in range(7):
+            if i == 6:
+                action_dict["left_left_carriage_joint.pos"] = left_arm_action[i]
+                action_dict["right_left_carriage_joint.pos"] = right_arm_action[i]
+            else:
+                action_dict[f"left_joint_{i}.pos"] = left_arm_action[i]
+                action_dict[f"right_joint_{i}.pos"] = right_arm_action[i]
+        self.robot.send_action(action_dict)
+
+    def run_episode(self, max_steps: int = 1000, task_prompt: str = "look down"):
+        """Run a single episode of policy execution."""
+        logger.info(f"Starting episode with prompt: '{task_prompt}'")
+        self.robot.configure()
+        self.episode_step = 0
+        self.action_chunk_idx = 0
+        self.current_action_chunk = None
+        self.is_running = True
+
+        while self.is_running and self.episode_step < max_steps:
+            start_loop_time = time.perf_counter()
+            observation_dict = self.robot.get_observation()
+
+            # Extract joint positions from observation
+            joint_pos_keys = [k for k in observation_dict.keys() if k.endswith('.pos')]
+            joint_positions = np.array([observation_dict[k] for k in joint_pos_keys])
+
+
+            # Transform and resize images from all cameras
+            cameras = ['top', 'bottom', 'right', 'left']
+            for cam in cameras:
+                image_hwc = observation_dict[cam]
+                image_resized = cv2.resize(image_hwc, (224, 224))
+                image_chw = np.transpose(image_resized, (2, 0, 1))
+                observation_dict[cam] = image_chw
+
+            # Create observation for policy to follow the ALOHA format
+            observation = {
+                "state": joint_positions,
+                "images": {
+                    "cam_high": observation_dict['top'],
+                    "cam_low": observation_dict['bottom'],
+                    "cam_right_wrist": observation_dict['right'],
+                    "cam_left_wrist": observation_dict['left'],
```

Comment:
What's the reasoning behind this mismatch? we're following the aloha format here, but can't we use the same names from the beginning?